src/main/kotlin/blackjack/model/player/Dealer.kt

```diff
@@ -2,30 +2,39 @@ package blackjack.model.player
 
 import blackjack.model.card.Card
 import blackjack.model.card.CardDeck
+import blackjack.model.game.Rank
 import blackjack.model.state.State
 import blackjack.model.state.playState.gameState.Bust
 import blackjack.model.state.playState.gameState.Hit
 
-data class Dealer(val cards: CardDeck = CardDeck()) {
-    var state: State = Hit(cards)
+data class Dealer(var state: State) {
+    constructor(cards: CardDeck = CardDeck()) : this(Hit(cards))
 
     fun draw(card: Card) {
         this.state = state.draw(card)
     }
 
-    fun stay() {
-        this.state = state.stay()
-    }
-
-    fun isBust() = state is Bust
-
     fun isOverSixTeen() = score() <= SIXTEEN
 
     fun cards(): List<Card> {
-        return state.cards().deck
+        return state.cards()
     }
 
-    fun score() = state.cards().calculateScore()
+    fun score() = state.calculateScore()
+
+    fun isWin(player: Player): Rank {
+        return when {
+            isWinningCondition(player) -> Rank.WIN
+            isDrawCondition(player) -> Rank.DRAW
```

Comment:
굳이 변경안하셔도 되지만 확장함수를 이용하여 아래와 같이 표현하는 것도 하나의 방법입니다.

https://kotlinlang.org/docs/extensions.html

```suggestion
            player.isWinningCondition() -> Rank.WIN
            player.isDrawCondition() -> Rank.DRAW
```